docs/source/developer_guides/lora.md

```diff
@@ -53,6 +53,26 @@ lora_config = LoraConfig(..., init_lora_weights="loftq", loftq_config=loftq_conf
 peft_model = get_peft_model(base_model, lora_config)
 ```
 
+An easier way to apply LoftQ initialization is to use the convenience function `replace_lora_weights_loftq` from PEFT. This takes the quantized PEFT model as input and replaces the LoRA weights in-place with their LoftQ-initialized counterparts.
```

Comment:
I would also maybe explain the code snippet below a bit more. For example, adding a sentence explaining the `BitsAndBytesConfig` a bit more (ie, it comes from the Transformers library and include a link to the config class or to the [docs](https://huggingface.co/docs/transformers/quantization#8-bit).

```suggestion
An easier way to apply LoftQ initialization is to use the convenience function `replace_lora_weights_loftq`. This takes the quantized PEFT model as input and replaces the LoRA weights in-place with their LoftQ-initialized counterparts.
```